twine/repository.py

```diff
@@ -25,7 +25,14 @@
 import twine
 from twine import package as package_file
 
-KEYWORDS_TO_NOT_FLATTEN = {"gpg_signature", "attestations", "content"}
+KEYWORDS_TO_NOT_FLATTEN = {"gpg_signature", "attestations", "content", "keywords"}
+
+# Map ``packaging.metadata.RawMetadata`` fields to package upload form fields.
```

Comment:
Is it not possible to simply use the same singular or plural keyword to match? Why are `classifiers` not affected by this need to rename?

Reply:
The field names in `packaging.metadata.RawMetadata` are not under the control of `twine` and using `packaging.metadata.RawMetadata` is implicit in using `packaging.metadata` for parsing and validating metadata. Somewhere the rename needs to happen. Renaming here makes the code easier everywhere else. The `packaging` maintainers have reasons for naming the fields like this, see the `packaging` documentation for their reasoning.